custodial-copy-reconciler/src/main/scala/uk/gov/nationalarchives/reconciler/Database.scala

```diff
@@ -22,6 +22,7 @@ trait Database[F[_]]:
   def writeToPreservicaCOsTable(cosInPS: Chunk[CoRow]): F[Unit]
   def writeToOcflCOsTable(expectedCosInPS: Chunk[CoRow]): F[Unit]
   def findAllMissingCOs(): F[Result]
+  def deleteFromTables(): F[Unit]
```

Comment:
Maybe `clearTables` or `emptyTables`?

Reply:
`deleteFromTables` is a decent enought description of what it does.